inspirehep/modules/disambiguation/logic.py

```diff
@@ -0,0 +1,428 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Inspire.
+# Copyright (C) 2016 CERN.
+#
+# Inspire is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Inspire is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Inspire; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Logic of Disambiguation module."""
+
+from __future__ import absolute_import, division, print_function
+
+from operator import add
+
+import celery
+from celery.utils.log import get_task_logger
+from flask import current_app
+
+from inspirehep.modules.disambiguation.records import create_author
+from inspirehep.modules.disambiguation.search import create_record, \
+    get_signature
```

Comment:
Thanks, anyway I would appreciate straightforward comments.

Imports using `\` are following `PEP-8`.
I mean I cannot blame myself for not putting whitespace after `\` and thus having imports working correctly.